packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -987,6 +999,33 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
         }
     }
 
+    protected async configureDisplayLanguage(): Promise<void> {
```

Comment:
Running this in Electron and choosing the German option gave me a white window that never loaded. No untoward logs on the backend, though, so I'm not exactly sure what happened.

Reply:
I figured this one out. It seems like `String.prototype.replaceAll` wasn't available in the electron frontend. I used it in the translation process which led to an error on startup. Replacing it with `String.prototype.replace` and a regex fixed the issue.

Reply:
That seems to have fixed it. I'm now able to Configure Display Lanuguage / Sprache Wählen freely in Electron. 👍